### PR TITLE
Feature/request new version

### PR DESCRIPTION
**Pull request by doombot-exe**
fix dockerfile for installing perl module to handle random number generation

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,9 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 MAINTAINER Rederly
 
 WORKDIR /usr/app
+ARG DEBIAN_FRONTEND=noninteractive
```
Comment (red-tt-bear) [inline Dockerfile:5]: What do these do?
Why are we setting TZ to new york?

File: public/version [hunk 2]
```diff
@@ -1 +1 @@
-1.1.1
\ No newline at end of file
+1.2
```
Comment (red-tt-bear) [inline public/version:1]: Not for now but I prefer keeping this three digits
1.2.0

File: Dockerfile (hunk 1 above)
Comment (doombot-exe) [reply Dockerfile:5]: https://anonoz.github.io/tech/2020/04/24/docker-build-stuck-tzdata.html

This is temporary - an end-run around getting 20.04 to build in docker

Comment (red-tt-bear) [reply Dockerfile:5]: Gotcha, putting that in a comment in the dockerfile would be helpful in case we forget in the future

**Review by red-tt-bear — APPROVED** [review]

**Pull request merged**